monkestation/code/datums/brain_damage/magic.dm

```diff
@@ -43,6 +43,7 @@
 	// Not even nullspace will keep it at bay.
 	for (var/obj/effect/client_image_holder/stalker_phantom/stalker in stalkers)
 		if(!stalker || !stalker.loc || stalker.z != owner.z)
```

Comment:
stalkers will be qdeled and recreated every tick if the victim's loc isn't a turf

non-turf loc will result in them being nullspaced: https://github.com/Monkestation/Monkestation2.0/blob/58889948bcb00ef89bcd2da85c6cab0f55b5fd1e/monkestation/code/datums/brain_damage/magic.dm#L56

which will cause them to be qdeled and recreated the next life tick

even without the hard dels, that's still going to cause unnecessary load on the server